gensim/summarization/summarizer.py

```diff
@@ -211,6 +212,11 @@ def summarize(text, ratio=0.2, word_count=None, split=False):
 
     most_important_docs = summarize_corpus(corpus, ratio=ratio if word_count is None else 1)
 
+    # If couldn't get important docs, the algorithm ends.
+    if not most_important_docs:
+        logger.warning("Couldn't get relevant sentences.")
+        return
```

Comment:
Raising an exception better? I'm not sure whether this is an error state, or just a warning.

Many people don't have logging enabled, and the docstring implies the result of this function is a string (not `None`).

Reply:
I don't think this is an error. Perhaps it makes even more sense to actually return the entire text as the summary wasn't possible, but this will break compatibility with the old behavior.

Regarding the docstring, the method returns a string or a list if the `split` parameter was set to true, so perhaps the best thing to do is:

```
if not most_important_docs:
    logger.warning("Couldn't get relevant sentences.")
    return [] if split else ""
```

Reply:
That looks like a good solution 👍 
CC @menshikh-iv